modules/distribution/src/repository/resources/conf/templates/repository/conf/identity/identity.xml.j2

```diff
@@ -917,79 +1114,111 @@
     This property restricts federated user association done through UserProfileAdmin admin service.
```

Comment:
review  [L913](https://github.com/wso2/product-is/pull/4572/files#diff-8439dcc4cbbf52386455006c761214b0L913) 

review [L773](https://github.com/wso2/product-is/pull/4572/files#diff-8439dcc4cbbf52386455006c761214b0L773)